packages/react-core/src/components/Page/Page.tsx

```diff
@@ -253,25 +276,38 @@ export class Page extends React.Component<PageProps, PageState> {
       nav = <div className={css(styles.pageMainNav)}>{tertiaryNav}</div>;
     }
 
-    let crumb = null;
-    if (breadcrumb && isBreadcrumbWidthLimited) {
-      crumb = (
-        <section className={css(styles.pageMainBreadcrumb, styles.modifiers.limitWidth)}>
-          <div className={css(styles.pageMainBody)}>{breadcrumb}</div>
-        </section>
-      );
-    } else if (breadcrumb) {
-      crumb = <section className={css(styles.pageMainBreadcrumb)}>{breadcrumb}</section>;
-    }
+    const crumb = (
+      <section
+        className={css(
+          styles.pageMainBreadcrumb,
+          isBreadcrumbWidthLimited && styles.modifiers.limitWidth,
+          formatBreakpointMods(
+            breadcrumbProps?.stickyOnBreakpoint,
+            styles,
+            'sticky-',
+            getVerticalBreakpoint(height),
+            true
+          )
+        )}
+      >
+        {isBreadcrumbWidthLimited ? <div className={css(styles.pageMainBody)}>{breadcrumb}</div> : breadcrumb}
+      </section>
+    );
 
     const isGrouped = isTertiaryNavGrouped || isBreadcrumbGrouped || additionalGroupedContent;
 
     const group = isGrouped ? (
-      <PageGroup {...groupProps}>
+      <div
```

Comment:
I am not understanding why we are not using the `<PageGroup>` component anymore.

Reply:
@tlabaj Using `PageGroup` here creates a circular dependency between `Page` and `PageGroup`, which causes an error in the build.

Using the syntax above aligns with the way the [`breadcrumb` was already implemented in the file](https://github.com/patternfly/patternfly-react/pull/7764/files/466d9540a82d34498f53fb8dc0e1998e985742ae#diff-33f100e4656af51b92b0c2540a3ad8066a7384595d7b8d399fdbd448515c46ccR301-R315), i.e. declaring the css class on `<section>` rather than using `<PageBreadcrumb>`, which was likely intentional to prevent a circular dependency there!